_posts/2016-06-30-state-in-scalable-architectures.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+name: State-in-scalable-architectures
+title: 'State in scalable architectures'
+date: 2016-06-30 00:20:00 +00:00
+author: Felipe Fernández
+image:
+    src: /assets/img/custom/blog/state.jpg
+    attribution:
+      text: Untitled by Daida Medina
+      href: https://www.instagram.com/p/8TdviFHo-g/?taken-by=thelastfreeusername
```

Comment:
The picture was taken by my girlfriend, I'll ask her to provide proper licenses for her pictures: http://i-am-cc.org/